src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -0,0 +1,244 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+import createWrapper from '../../../lib/components/test-utils/selectors';
+import { drawerIds as drawerIdObj } from '../../../lib/dev-pages/pages/app-layout/utils/drawer-ids';
+import { viewports } from './constants';
+
+import visualRefreshStyles from '../../../lib/components/app-layout/visual-refresh/styles.selectors.js';
+import toolbarStyles from '../../../lib/components/app-layout/visual-refresh-toolbar/toolbar/styles.selectors.js';
+import tooltipStyles from '../../../lib/components/internal/components/tooltip/styles.selectors.js';
+
+const wrapper = createWrapper().findAppLayout();
+
+class AppLayoutDrawersPage extends BasePageObject {
+  async getElementCenter(selector: string) {
+    const targetRect = await this.getBoundingBox(selector);
+    const x = Math.round(targetRect.left + targetRect.width / 2);
+    const y = Math.round(targetRect.top + targetRect.height / 2);
+    return { x, y };
+  }
+
+  /**
+   * This holds the pointer down for an longer than the buttonDownOnElement
+   * @param selector
```

Comment:
it explains why the function is needed compared to a the button down. There is a grammar error in there I will fix, but if someone later visits the code and wonders why, this will explain it

Reply:
no longer needed